### ISSUE TITLE
replay-sorcery.service: Failed with result 'core-dump'.

### ISSUE THREAD

**Issue by UT66** [matanui159/ReplaySorcery#157]
It worked fine before but I wanted to try hardwareaccel, so I enabled that in the config file and now I get this error.

Also logs:
- Logs begin at Mon 2022-06-20 20:39:06 CEST. --
jun 22 22:54:28 utrinity-PC replay-sorcery[2629]:  - /home/utrinity/ReplaySorcery/src/main.c:84 (mainStep)
jun 22 22:54:28 utrinity-PC replay-sorcery[2629]:  - /home/utrinity/ReplaySorcery/src/main.c:207 (main)
jun 22 22:54:28 utrinity-PC replay-sorcery[2629]:  - ../csu/libc-start.c:308 (__libc_start_main)
jun 22 22:54:28 utrinity-PC systemd[1755]: replay-sorcery.service: Main process exited, code=dumped, status=6/ABRT
jun 22 22:54:28 utrinity-PC systemd[1755]: replay-sorcery.service: Failed with result 'core-dump'.
jun 22 22:54:29 utrinity-PC systemd[1755]: replay-sorcery.service: Scheduled restart job, restart counter is at 5.
jun 22 22:54:29 utrinity-PC systemd[1755]: Stopped ReplaySorcery: an open-source, instant-replay solution for Linux..
jun 22 22:54:29 utrinity-PC systemd[1755]: replay-sorcery.service: Start request repeated too quickly.
jun 22 22:54:29 utrinity-PC systemd[1755]: replay-sorcery.service: Failed with result 'core-dump'.
jun 22 22:54:29 utrinity-PC systemd[1755]: Failed to start ReplaySorcery: an open-source, instant-replay solution for Linux..

**Comment by matanui159** [comment]
Can you please share more of the logs? I can't get any information from this

**Comment by UT66** [comment]
Yes I would love to share more logs I just don't know how haha sorry, bit of a linux noob.
So if you could tell me what command to put then I'll gladly share the logs :)

**Comment by trishmapow** [comment]
@UT66 You can use `journalctl --user-unit replay-sorcery`. @matanui159 I have a similar issue with core dump, here's the relevant log:
```
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]: Created X11 controller
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]: Segment violation
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /home/decoda/ReplaySorcery/src/log.c:76 (logSignal)
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /build/glibc-SzIz7B/glibc-2.31/signal/../sysdeps/unix/sysv/linux/x86_64/sigaction.c:0 ((null))
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /home/decoda/ReplaySorcery/src/encoder/ffenc.c:104 (ffmpegEncoderSendFrame)
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /home/decoda/ReplaySorcery/src/audio/../encoder/encoder.h:34 (rsEncoderSendFrame)
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /home/decoda/ReplaySorcery/src/main.c:84 (mainStep)
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - /home/decoda/ReplaySorcery/src/main.c:207 (main)
Jul 01 16:26:09 decoda-CCM-1 replay-sorcery[2103]:  - ../csu/libc-start.c:308 (__libc_start_main)
```

**Comment by UT66** [comment]
> @UT66 You can use `journalctl --user-unit replay-sorcery`. 
Thank you!

**Comment by UT66** [comment]
I get this:

jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: [h264_vaapi @ 0x55a2176a3b40] Driver does not support packed sequence headers, but a global header is requested.
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: [h264_vaapi @ 0x55a2176a3b40] No global header will be written: this may result in a stream which is not usable for some purposes (e.g. not muxable to s>
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: [h264_vaapi @ 0x55a2176a3b40] Quality attribute is not supported: will use default quality level.
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: Created VA-API encoder
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: PulseAudio server: pulseaudio 13.99.1
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: Connecting to Pulse Audio device: alsa_output.usb-Kingston_HyperX_Quadcast_4110-00.analog-stereo.monitor...
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: Created PulseAudio device
jun 30 17:14:31 utrinity-PC replay-sorcery[2714]: Created X11 controller
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]: Segment violation
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /home/utrinity/ReplaySorcery/src/log.c:76 (logSignal)
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /build/glibc-SzIz7B/glibc-2.31/signal/../sysdeps/unix/sysv/linux/x86_64/sigaction.c:0 ((null))
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /home/utrinity/ReplaySorcery/src/encoder/ffenc.c:104 (ffmpegEncoderSendFrame)
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /home/utrinity/ReplaySorcery/src/audio/../encoder/encoder.h:34 (rsEncoderSendFrame)
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /home/utrinity/ReplaySorcery/src/main.c:84 (mainStep)
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - /home/utrinity/ReplaySorcery/src/main.c:207 (main)
jun 30 17:14:32 utrinity-PC replay-sorcery[2714]:  - ../csu/libc-start.c:308 (__libc_start_main)
jun 30 17:14:32 utrinity-PC systemd[1833]: replay-sorcery.service: Main process exited, code=dumped, status=6/ABRT
jun 30 17:14:32 utrinity-PC systemd[1833]: replay-sorcery.service: Failed with result 'core-dump'.
jun 30 17:14:32 utrinity-PC systemd[1833]: replay-sorcery.service: Scheduled restart job, restart counter is at 5.
jun 30 17:14:32 utrinity-PC systemd[1833]: Stopped ReplaySorcery: an open-source, instant-replay solution for Linux..
jun 30 17:14:32 utrinity-PC systemd[1833]: replay-sorcery.service: Start request repeated too quickly.
jun 30 17:14:32 utrinity-PC systemd[1833]: replay-sorcery.service: Failed with result 'core-dump'.
jun 30 17:14:32 utrinity-PC systemd[1833]: Failed to start ReplaySorcery: an open-source, instant-replay solution for Linux..
